test/test_bin/test_sentry_usage_alert.py

```diff
@@ -23,24 +23,54 @@ def test_main_smoke_test(self, mock_sentry_client: MagicMock, mock_slack_service
     @patch.dict(os.environ, {"SLACK_TOKEN": "token"})
     @patch.dict(os.environ, {"PERIOD_IN_DAYS": "1"})
     @patch.dict(os.environ, {"USAGE_THRESHOLD": "20"})
-    def test_sends_notifications_to_slack_when_usage_above_threshold(self, mock_sentry_client: MagicMock,
-                                                                     mock_slack_service: MagicMock):
+    def test_sends_notifications_to_slack_when_usage_above_threshold(
+        self, mock_sentry_client: MagicMock, mock_slack_service: MagicMock
+    ):
         mock_sentry_client.return_value.get_usage_total_for_period_in_days.return_value = 10000000, START_TIME, END_TIME
         sentry_usage_alert.main()
         mock_slack_service.return_value \
-            .send_error_usage_alert_to_operations_engineering \
-            .assert_called_with(1, UsageStats(total=10000000,
-                                              max_usage=129032,
-                                              percentage_of_quota_used=77.50015500031,
-                                              start_time=START_TIME,
-                                              end_time=END_TIME), 0.2)
-        mock_slack_service.return_value \
-            .send_transaction_usage_alert_to_operations_engineering. \
-            assert_called_with(1, UsageStats(total=10000000,
-                                             max_usage=967741,
-                                             percentage_of_quota_used=10.333343322231878,
-                                             start_time=START_TIME,
-                                             end_time=END_TIME), 0.2)
+            .send_usage_alert_to_operations_engineering \
+            .assert_has_calls(
+                [
+                    call(
+                        1,
+                        UsageStats(
+                            total=10000000,
+                            max_usage=129032,
+                            percentage_of_quota_used=77.50015500031,
+                            start_time='2023-06-08T00:00:00Z',
```

Comment:
What was the reason for removing the START_TIME and END_TIME consts?

Reply:
I didn't

Reply:
Ah I see - the one I copy pasted from to create mine must've had the start and date times not as a constant; I can put that back

Reply:
Actually it was cos I used the `pytest` output to paste fromt he Actual to the Expected - and that had the dates in full.